src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -44,11 +44,14 @@
         <div class="row">
             <div class="col-md-8">
                 <div class="row">
-                    <i:for type="sirius.biz.jobs.params.Parameter" var="param" items="job.getParameters()">
+                    <i:for type="sirius.biz.jobs.params.Parameter" var="param" items="job.getVisibleParameters(context)">
                         <i:dynamicInvoke template="@param.getTemplateName()" param="@param" context="@context"/>
                     </i:for>
                 </div>
             </div>
+            <i:for type="sirius.biz.jobs.params.Parameter" var="param" items="job.getInvisibleParameters(context)">
```

Comment:
add this block in the interactive-job html.pasta as well?

Reply:
Covered with new redesign